frontend/public/manifest.json

```
@@ -34,8 +36,14 @@
   "scope": "/",
   "start_url": "/pwa/start?utm_source=pwa",
   "display": "standalone",
+  "display_override": ["window-controls-overlay"],
+  "orientation": "natural",
   "theme_color": "#ffc800",
   "background_color": "#a09b87",
+  "launch_handler": {
+    "client_mode": ["auto"]
+  },
+  "handle_links": "preferred",
   "share_target": {
     "action": "/shared-content",
```

Due to https://github.com/GoogleChromeLabs/bubblewrap/issues/698 we may need to use an absolute path here

```suggestion
    "action": "https://tournesol.app/shared-content",
```